konrad/convection.py

```diff
@@ -360,7 +360,8 @@ def update_convective_top(self, T_rad, T_con, p, timestep=0.1, lim=0.2):
 
             # index of the uppermost level where convection is applied (HardAdj)
             # and causes warming (RlxAdj)
-            contop_index = np.argmin(convective_heating > 0)
+            contop_index = (len(convective_heating) -
```

Comment:
If you simply want to "search the other way around" you could use
```python
contop_index = np.argmax(y <= 0)
```
which should be equivalent to your implementation.

Anyhow, I have the feeling that we should update the comment to explain **why** we are doing what we are doing. Especially if the order is important for some model configurations that are not obvious here.

Reply:
That doesn't quite fit what I want. If I have a convective heating profile like
`heating = np.array([2, -1, 1, 2, 3, 0, 0, 0])`
then I want the first zero after which all other entries are zero. Maybe you can think of a nicer way to express that in code? But anyway, I will improve the comment.

Reply:
Okay, I see. Unfortunately, I cannot think of a nicer implementation. But I guess with an updated comment it is just fine